app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    args = command.split(" ")
+
+    if len(args) == 3:
+        cmd, file1, file2 = args
+
+        if cmd == "mv" and file1 != file2:
+            dir_file2, name_file2 = os.path.split(file2)
+
+            ls_dir_file2 = dir_file2.split("/")
+            item = list()
+            path_file2 = ""
+
+            for elem in ls_dir_file2:
+                item.append(elem)
+                path_file2 = os.path.join(*item)
+
+                try:
+                    os.mkdir(path_file2)
+                except WindowsError:
+                    continue
```

Comment:
You can use `os.makedirs` with the argument `exist_ok=True` to avoid exception here